compiler-rt/lib/ctx_profile/CtxInstrProfiling.cpp

```diff
@@ -272,6 +272,8 @@ void setupContext(ContextRoot *Root, GUID Guid, uint32_t NumCounters,
 
 ContextRoot *FunctionData::getOrAllocateContextRoot() {
   auto *Root = CtxRoot;
+  if (mustNotBeRoot(Root))
```

Comment:
```suggestion
  if (isLegalRoot(Root))
```
Then below you won't need a double negative (`!mustNotBeRoot()`).  Feel free to ignore, just seems easier to parse.